lib/scope.js

```diff
@@ -87,7 +87,7 @@ ScopeDefinition.prototype.related = function(receiver, scopeParams, condOrRefres
   cb = cb || utils.createPromiseCallback();
 
   if (!self.__cachedRelations || self.__cachedRelations[name] === undefined ||
-      actualRefresh) {
+    actualRefresh) {
```

Comment:
This change makes it difficult to see where the condition ends and where the body of the `if` branch starts. It's also unrelated to fixing a missing option argument, I am proposing to revert it, together with other white-space only changes below.

Alternatively, if we want to fix this, then let's do it properly please - see http://loopback.io/doc/en/contrib/style-guide.html#indentation-of-multi-line-expressions-in-if